### ISSUE TITLE
Input fields..

### ISSUE THREAD

**Issue by nicklasb** [nakupanda/bootstrap3-dialog#114]
I don't know if this is within the scope of this project, but is it possible to add some kind of control of the middle area of the controller? So that it becomes possible to add any markup one would like there?
I am thinking of uses like login dialogs, adding new items. Stuff like that. 
Also wizards could then can be quite easily created by chaining several dialogs.

**Comment by nakupanda** [comment]
"uses like login dialogs, adding new items. Stuff like that, with input. "

This should be possible and easily be done.

I'm not sure if I got your mind correctly, if you meant 'Creating facny contents including loing form inside a dialog', that's possible. Or it's something else?

**Comment by nicklasb** [comment]
Yeah, I would think so. It should not be much more than a "content"-parameter or something. 
I don't thing that the component in general would need to have any more functionality.

Yes, basically, and perhaps also get access to the resulting div or something in the output, perhaps there can be a second result parameter if the content is used. I would certainly find it very useful.
